source/common/ratelimit/ratelimit.proto

```diff
@@ -1,98 +0,0 @@
-syntax = "proto3";
```

Comment:
Did you search the docs to make sure nothing refers to this file currently? I was looking at this recently and I thought there might be a reference somewhere?

Reply:
Found one and fixed in c79ef8e.

Reply:
Sorry docs is still failing. I would try to generate the docs locally and check them (`docs/build.sh`).